net/Socket.cpp

```diff
@@ -877,8 +903,6 @@ std::shared_ptr<Socket> ServerSocket::accept()
     // Accept a connection (if any) and set it to non-blocking.
     // There still need the client's address to filter request from POST(call from REST) here.
 #if !MOBILEAPP
-    assert(_type != Socket::Type::Unix);
-
```

Comment:
How does this work for client addresses ? I assume that sockaddr_in6 is big enough for whatever a unix socket whacks in clientInfo but is it ? =)

Reply:
Did you wrongly quoted the code? With this assert the code will stop